framework/PostProcessors/FTgate.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on April 30, 2018
+
+@author: mandd
+"""
+
+from __future__ import division, print_function , unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+
+#Internal Modules---------------------------------------------------------------
+import MessageHandler
+from utils import utils
+from utils import xmlUtils as xmlU
+#Internal Modules End-----------------------------------------------------------
+
+#External Modules---------------------------------------------------------------
+import numpy as np
+import xml.etree.ElementTree as ET
+import copy
+import itertools
+from collections import OrderedDict
+#External Modules End-----------------------------------------------------------
+
+class FTgate():
```

Comment:
no inheritance?

Reply:
Add ```object``` for this class. however, this is a utils function that is used by both FTImporter post processor and FTModel external model plugin. Since messageHandler is not available for External Model Plugin, we could not pass the messageHandler for this class.